src/sentry/api/endpoints/rule_snooze.py

```diff
@@ -0,0 +1,96 @@
+import datetime
+
+from rest_framework import serializers, status
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import audit_log, features
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.project import ProjectAlertRulePermission, ProjectEndpoint
+from sentry.api.exceptions import ConflictError
+from sentry.api.serializers import serialize
+from sentry.api.serializers.rest_framework.base import CamelSnakeSerializer
+from sentry.incidents.models import AlertRule
+from sentry.models import Rule, RuleSnooze
+
+
+class RuleSnoozeSerializer(CamelSnakeSerializer):
+    user_id = serializers.IntegerField(required=False, allow_null=True)
```

Comment:
i missed the original review on the model, but why make this one a `User` instead of `Actor` like `AlertRule` and `Rule` do?

https://github.com/getsentry/sentry/blob/b67df8ce3aa7590658f3eb49afceb762dffb6bf5/src/sentry/incidents/models.py#L369

Reply:
It was discussed but the point of this feature is to ignore the alert for a particular user or just ignore it entirely, so we didn't see the usefulness of using actor to have the option of teams as well.

Reply:
makes sense, thanks!